go-controller/pkg/util/multi_network_test.go

```diff
@@ -337,6 +335,62 @@ func TestParseNetconf(t *testing.T) {
 `,
 			expectedError: fmt.Errorf("layer3 topology does not allow persistent IPs"),
 		},
+		{
```

Comment:
Is it worth it to add a test to default network and `IsPrimaryNetwork` ?

Reply:
good point! lmc if i can add one for this

Reply:
done.